src/requests/workspace.jl

```diff
@@ -14,7 +14,7 @@ function process(r::JSONRPC.Request{Val{Symbol("workspace/didChangeWatchedFiles"
         elseif change.type == FileChangeTypes["Deleted"] && !get_open_in_editor(server.documents[URI2(uri)])
             delete!(server.documents, URI2(uri))
 
-            publishDiagnosticsParams = PublishDiagnosticsParams(uri, Diagnostic[])
+            publishDiagnosticsParams = PublishDiagnosticsParams(uri, missing, Diagnostic[])
```

Comment:
It seems safe to not send the version here? Not sure...

Reply:
Possibly but shouldn't we just send it anyway?

Reply:
Well, it is a bit unclear whether a doc that has been deleted actually still has a version?